Exoft.Gamification.Api.Services/ReferenceBookService.cs

```diff
@@ -93,7 +93,7 @@ public async Task<bool> AddArticleAsync(CreateArticleModel articleModel)
                         var tempArt = chapter.Articles.LastOrDefault(d => d.UnitNumber != null);
                         maxUnitNumber = tempArt != null ? int.Parse(tempArt.UnitNumber.ToString().Split(".")[1]) : 0;
                     }
-                    catch{}
+                    catch { }
```

Comment:
seems like a potentially dangerous place here. do we log exceptions here or handle them somehow?